forte/data/ontology/ontology_code_generator.py

```diff
@@ -454,38 +461,82 @@ def parse_ontology_spec(self, json_file_path: str,
         # Extract imported json files and generate ontology for them.
         json_imports: List[str] = spec_dict.get("import_paths", [])
 
-        for import_file in json_imports:
-            import_json_file = utils.search_in_dirs(import_file,
+        # Get location of installed forte
+        forte_origin = importlib.util.find_spec('forte').origin
+        installed_forte_dir = os.path.dirname(os.path.dirname(forte_origin))
+
+        for user_import in json_imports:
+            installed_json_file = None
+            is_package = '.' in user_import and '/' not in user_import
+            if is_package:
+                # Check installed modules
+                try:
+                    import_origin = importlib.util.find_spec(user_import).origin
+
+                    delimiters = '# ', SOURCE_JSON_PFX, SOURCE_JSON_SFX
```

Comment:
These lines are using the command to locate the jsons, so worth add some comments for documentation purpose.

Reply:
Added extensive comments wherever required.